docs/user-guide/creating-application/config-maps.md

```diff
@@ -86,6 +86,14 @@ You can click on `YAML` or `GUI` to view the key and Value parameters of the Con
 
 You can click on `Save ConfigMap` to save the configMap.
 
+
+## Sub Path
+For multiple files mount at the same location you need to check sub path bool field, it will use the file name (key) as sub path. 
```

Comment:
To mount mutiple files from different configmap and secret at the same location you need to check sub path bool field. It will mount key as the file name.